routes/index.js

```diff
@@ -19,34 +19,69 @@ If not, see <http://www.gnu.org/licenses/lgpl.html>.
 
 'use strict';
 
-const path = require('path'),
+const config = require('config'),
+	ensureLoggedIn = require('connect-ensure-login').ensureLoggedIn,
+	LocalStrategy = require('passport-local').Strategy,
+	path = require('path'),
+	passport = require('passport'),
+	session = require("express-session"),
 	router = require('express').Router();
 
-const db = require('../db');
+// just for testing
+const guest = {
+	email: 'nirdizati',
```

Comment:
these credentials seem to be ignored. I can log in with any combination of username-password

Reply:
yes, it does not suppose to work yet